diagnostic.md

```diff
@@ -6,70 +6,70 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What are two ways of telling rails where to route a GET request to '/books/4'?
 
 ```rb
-// your response here
+@books = Book.find(params[4])
 ```
 
 In what file does the above code go?
 
 ```md
-// your response here
+In the routes.rb file
 ```
 
 In your own words, what does the command `bundle exec rake db:migrate` do?
 
 ```md
-// your response here
+This command runs migrations that have not been run yet, specifically for new databases or controllers.
```

Comment:
It executes any code found within the Active Record migration files in db/migrate. The migrations make it so that we don't have to modify our database directly.